src/Microsoft.VisualStudio.Composition/NullableBool.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.VisualStudio.Composition
+{
+    using System;
+
+    /// <summary>
+    /// A nullable boolean value with atomic reads and writes.
+    /// </summary>
```

Comment:
Future readers may appreciate a remark such as: `Nullable<bool>` is larger than the machine's word size and is subject to tearing, which can result in invalid state during data races.

Reply:
Done.